### PR TITLE
Fix a couple of nits while running through for first time.

### PR DESCRIPTION
**Pull request by Billy99**
Signed-off-by: Billy McFall <22157057+Billy99@users.noreply.github.com>

### CODE REVIEW

File: docs/book/start/dropping-packets.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Here's how the userspace code looks:
 ## Running the program
 
 ```console
-$ cargo xtask run
+$ cargo xtask run -- target/bpfel-unknown-none/debug/myapp
```
Comment (vadorovsky) [inline docs/book/start/dropping-packets.md:63]: That shouldn't be needed? For simple examples, `cargo xtask run` should be enough and the bytes of eBPF program should be included in the binary.

Comment (Billy99) [reply docs/book/start/dropping-packets.md:63]: OK, it panic'd on me, so not sure what I was doing wrong.

Comment (Billy99) [reply docs/book/start/dropping-packets.md:63]: I guess if we remove the `None => panic!("no path provided"),` on line 18, it wold work.

Comment (dave-tucker) [reply docs/book/start/dropping-packets.md:63]: The root problem is here:
https://github.com/aya-rs/book/blob/51550af86e790e8125d161c0b36e7b11800ba645/examples/myapp-02/myapp/src/main.rs#L16-L19

It should use `include_bytes_aligned!` like here:
https://github.com/aya-rs/book/blob/926a81fba12859d3935d963f242d6379e9215ecf/examples/myapp-01/myapp/src/main.rs#L34-L40

And the only arg provided should be the path.

Comment (dave-tucker) [reply docs/book/start/dropping-packets.md:63]: @Billy99 if you don't want to tackle updating those examples to use `include_bytes_aligned!` like above, if you back out this change I'll update the examples later today

Comment (Billy99) [reply docs/book/start/dropping-packets.md:63]: I'll give it a shot in a little while. I was also planning on a 2nd PR to address the template change from `StructOpt` to `clap` (if that's OK).

Comment (Billy99) [reply docs/book/start/dropping-packets.md]: @dave-tucker My latest commit addresses this.

On a side note, I used `../../target/bpfel-unknown-none/release/myapp`, similar to "Hello XDP!". However, based on the instructions, only `.../debug/...` is created. In a follow-up PR, I will update this based on the latest aya-template, which looks like:
```
    #[cfg(debug_assertions)]
    let mut bpf = Bpf::load(include_bytes_aligned!(
        "../../target/bpfel-unknown-none/debug/myapp"
    ))?;
    #[cfg(not(debug_assertions))]
    let mut bpf = Bpf::load(include_bytes_aligned!(
        "../../target/bpfel-unknown-none/release/myapp"
    ))?;
```

**Pull request merged**